lib/sentry/logger_handler.ex

```diff
@@ -44,6 +44,15 @@ defmodule Sentry.LoggerHandler do
       If set to `:all`, all metadata will be included.
       """
     ],
+    tags: [
+      type: {:or, [{:list, :atom}, {:in, [:all]}]},
+      default: [],
+      type_doc: "list of `t:atom/0`, or `:all`",
+      doc: """
+      Use this to include logger metadata as tags in reports. If it's a list of keys, metadata
+      in those keys will be added as tags to the event. If set to `:all`, all metadata will be included as tags.
```

Comment:
I don't think we can include **all** metadata as tags because some metadata would be really worthless as tags (`:line` as an example). We should discourage that behavior from users I think. Thoughts?

Reply:
Yeah, I agree. I'll remove the `:all`

Reply:
What are your thoughts on `tags` vs `tags_from_metadata` here? I'm OK with either given it lives in `runtime.exs` and isn't repeated often

Reply:
`tags_from_metadata` is more explicit I think.